DMTR-102.tex

```diff
@@ -44,183 +40,377 @@ \section{Introduction}
 \subsection{Objectives}
 \label{sect:objectives}
 
-Testing for the raw image from the DAQ to being written to the archive service at the LDF
-This test activity demonstrates the successful execution of the major components of the storing of raw image files from the DAQ to the DBB and then visualized through the LSST science platform (LSP).   This is at a very small scale and it to test the components of a proof of concept that they all work together.  
+Testing for the raw image from the DAQ to being written to the archive
+service at the LDF\\
+This test activity demonstrates the successful execution of the major
+components of the storing of raw image files from the DAQ to the DBB and
+then visualized through the LSST science platform (LSP). ~ This is at a
+very small scale and it to test the components of a proof of concept
+that they all work together. ~\\[2\baselineskip]It will demonstrate
 
-It will demonstrate
 \begin{itemize}
-\item Data can be transferred into the lsst data facility (LDF).   
-\item The raw data images can be ingested by the DBB along with some metadata about the image and providence.  
-\item The data files  can then be retrieved by the LSP
+\tightlist
+\item
+  Data can be transferred into the lsst data facility (LDF). ~~
```

Comment:
So many spurious `~`s! I guess they are harmless, though.

Reply:
The autogeneration script is preserving extra spaces when provided. They are harmless. You can edit the text in the test cases/test plan/test cycles/test results in order to get rid of them.